src/System.Windows.Forms/src/System/Windows/Forms/TreeNode.cs

```diff
@@ -2132,7 +2134,7 @@ internal unsafe void UpdateImage()
         {
             mask = TVITEM_MASK.TVIF_HANDLE | TVITEM_MASK.TVIF_IMAGE,
             hItem = HTREEITEM,
-            iImage = Math.Max(0, ((ImageIndexer.ActualIndex >= tv.ImageList.Images.Count) ? tv.ImageList.Images.Count - 1 : ImageIndexer.ActualIndex))
+            iImage = Math.Max(0, ((ImageIndexer.ActualIndex >= tv.ImageList!.Images.Count) ? tv.ImageList.Images.Count - 1 : ImageIndexer.ActualIndex))
```

Comment:
I'm not sure if the intent was to throw if `ImageList` is null here.. Can you see if there are tests where we are expecting an exception for this if `ImageList` is null? If not, perhaps change to:
```suggestion
            iImage = Math.Max(
                0,
                tv.ImageList is { } imageList && ImageIndexer.ActualIndex >= imageList.Images.Count
                    ? imageList.Images.Count - 1
                    : ImageIndexer.ActualIndex)
```